UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/appsettings.json

```diff
@@ -1,31 +1,41 @@
 {
-    "Logging": {
-        "PathFormat": "Logs/UKHO.MaritimeSafetyInformation.Web-{Date}.txt",
-        "Json": true,
-        "LogLevel": {
-            "Default": "Information",
-            "Microsoft.AspNetCore": "Warning"
-        },
-        "ApplicationInsights": {
-            "LogLevel": {
-                "Default": "Information",
-                "Azure.Core": "Warning"
-            }
-        }
+  "Logging": {
+    "PathFormat": "Logs/UKHO.MaritimeSafetyInformation.Web-{Date}.txt",
+    "Json": true,
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
     },
-    "EventHubLoggingConfiguration": {
-        "MinimumLoggingLevel": "Warning",
-        "UkhoMinimumLoggingLevel": "Information",
-        "Environment": "",
-        "EntityPath": "",
-        "System": "MSI",
-        "Service": "Maritime Safety Information",
-        "NodeName": "Azure",
-        "ConnectionString": ""
-    },
-    "KeyVaultSettings": {
-        "ServiceUri": ""
-    },
-    "APPINSIGHTS_INSTRUMENTATIONKEY": "",
-    "AllowedHosts": "*"
+    "ApplicationInsights": {
+      "LogLevel": {
+        "Default": "Information",
+        "Azure.Core": "Warning"
+      }
+    }
+  },
+  "EventHubLoggingConfiguration": {
+    "MinimumLoggingLevel": "Warning",
+    "UkhoMinimumLoggingLevel": "Information",
+    "Environment": "",
+    "EntityPath": "",
+    "System": "MSI",
+    "Service": "Maritime Safety Information",
+    "NodeName": "Azure",
+    "ConnectionString": ""
+  },
+  "KeyVaultSettings": {
+    "ServiceUri": ""
+  },
+  "FileShareService": {
+    "BaseUrl": "https://filesqa.admiralty.co.uk",
+    "BusinessUnit": "TEST",
+    "ProductType": "Notices to Mariners",
+    "PageSize": "100",
+    "Start": "0"
```

Comment:
Please remove **values** assigned for BaseUrl, BusinessUnit, ProductType  from config files and provided them through pipeline.

Reply:
Values for BaseUrl, BusinessUnit, ProductType removed from config